src/pal/inc/pal_corefx.h

```diff
@@ -45,6 +45,31 @@ ForkAndExecProcess(
            int* stdoutFd,
            int* stderrFd);
 
+struct corefx_stat {
+    int64_t dev;     /* ID of device containing file */
+    int32_t mode;    /* protection */
+    int32_t uid;     /* user ID of owner */
+    int32_t gid;     /* group ID of owner */
+    int64_t size;    /* total size, in bytes */
+    int64_t atime;   /* time of last access */
+    int64_t mtime;   /* time of last modification */
+    int64_t ctime;   /* time of last status change */
+};
```

Comment:
The problem with 0 is that these are Unix timestamps so instead of 1/1/1 12:00:00AM you get the Unix Epoch.  I would much prefer the bool, if it's all the same to you.